web/templates/settings/account_settings.hbs

```diff
@@ -94,6 +94,12 @@
                   label_parens_text=settings_label.presence_enabled_parens_text
                   help_link="/help/status-and-availability"
                   prefix="user_"}}
+                {{> settings_checkbox
+                  setting_name="allow_private_data_export"
+                  is_checked=settings_object.allow_private_data_export
+                  label=settings_label.allow_private_data_export
+                  help_link="/help/export-your-organization#full-export-with-member-consent"
+                  }}
```

Comment:
Should we include a dev env guard to make it visible only in dev environment ? I think, there is no downside if it is visible to users before the feature is complete ?

Reply:
Yeah, I think it's fine to have it here before administrators have a UI component to do it.